packages/survey-creator-core/src/components/tabs/theme-plugin.ts

```diff
@@ -21,6 +21,13 @@ function getObjectDiffs(obj1: any, obj2: any = {}): any {
   return result;
 }
 
+export interface IPropertyGridSurveyCreatedEvent {
```

Comment:
I'd prefer to pass themeBuilder model and themeEditorSurvey object rather that a set of methods. These helper methpds will be available through the themeBuilder model object and user will have direct acces to themeEditorSurvey object.